scripts.js

```diff
@@ -34,6 +34,22 @@ function loadCSS(href) {
   document.head.appendChild(link);
 };
 
+/**
+ * Return the correct CMP integration ID based on the domain name
+ */
+function getOtDomainId() {
+  const domains = {
+    'adobe.com': '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
+    'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271',
+    'project-helix.page': '45a95a10-dff7-4048-a2f3-a235b5ec0492',
+    'helix-demo.xyz': 'ff276bfd-1218-4a19-88d4-392a537b6ce3',
+  };
+  const currentDomain = Object.keys(domains).find(domain => window.location.host.indexOf(domain) > -1);
+
+  //TODO: remove `-test` once integration is ready for production
```

Comment:
A second PR will be created to take care of this TODO and to update the `preconnect` items.